translations/en.json

```diff
@@ -135,6 +135,7 @@
             "edit": {
                 "add_share": "Add Share",
                 "enter_valid_folder_name": "Enter a valid folder name",
+                "invalid_folder_name_warning_msg": "Special characters not allowed. Max lenght is 128 characters.",
```

Comment:
I don't get why would you add this string in this PR

Reply:
this string has been dynamically generated by i18next while in watch mode, as the key and the default value were added in another pr without adding them to the translation file. 
I can remove the string, but it will keep on appearing until someone adds it